server/src/main/java/org/apache/gravitino/server/web/rest/SchemaOperations.java

```diff
@@ -126,10 +146,18 @@ public Response createSchema(
   @Produces("application/vnd.gravitino.v1+json")
   @Timed(name = "load-schema." + MetricNames.HTTP_PROCESS_DURATION, absolute = true)
   @ResponseMetered(name = "load-schema", absolute = true)
+  @AuthorizationExpression(
```

Comment:
We should have the privilege `USE_CATALOG`  first.

Reply:
fixed